show/main.py

```diff
@@ -2916,5 +2916,288 @@ def autorestart(container_name):
+        if intf in nbrs_data:
+            nbrs_data[intf].append(ip)
+        else:
+            nbrs_data[intf] = [ip]
+
+    table = []
+    for k, v in vnet_intfs.items():
+        v = natsorted(v)
+        header[0] = k
+        table = []
+        for intf in v:
+            if intf in nbrs_data:
+                for ip in nbrs_data[intf]:
+                    r = ["", ip, intf]
+                    table.append(r)
+        click.echo(tabulate(table, header))
+        click.echo("\n")
+
+@vnet.group()
+def routes():
+    """Show vnet routes related information"""
+    pass
+
+@routes.command()
+def all():
+    """Show all vnet routes"""
+    appl_db = swsssdk.SonicV2Connector()
+    appl_db.connect(appl_db.APPL_DB)
+
+    header = ['vnet name', 'prefix', 'nexthop', 'interface']
+
+    # Fetching data from appl_db for VNET ROUTES
+    vnet_rt_keys = appl_db.keys(appl_db.APPL_DB, "VNET_ROUTE_TABLE*")
+    vnet_rt_keys = natsorted(vnet_rt_keys) if vnet_rt_keys else []
+
+    table = []
+    for k in vnet_rt_keys:
+        r = []
+        r.extend(k.split(":", 2)[1:])
+        val = appl_db.get_all(appl_db.APPL_DB, k)
+        r.append(val.get('nexthop'))
+        r.append(val.get('ifname'))
+        table.append(r)
+
+    click.echo(tabulate(table, header))
+
+    click.echo("\n")
+
+    header = ['vnet name', 'prefix', 'endpoint', 'mac address', 'vni']
+
+    # Fetching data from appl_db for VNET TUNNEL ROUTES
+    vnet_rt_keys = appl_db.keys(appl_db.APPL_DB, "VNET_ROUTE_TUNNEL_TABLE*")
+    vnet_rt_keys = natsorted(vnet_rt_keys) if vnet_rt_keys else []
+
+    table = []
+    for k in vnet_rt_keys:
+        r = []
+        r.extend(k.split(":", 2)[1:])
+        val = appl_db.get_all(appl_db.APPL_DB, k)
+        r.append(val.get('endpoint'))
+        r.append(val.get('mac_address'))
+        r.append(val.get('vni'))
+        table.append(r)
+
+    click.echo(tabulate(table, header))
+
+@routes.command()
+def tunnel():
+    """Show vnet tunnel routes"""
+    appl_db = swsssdk.SonicV2Connector()
+    appl_db.connect(appl_db.APPL_DB)
+
+    header = ['vnet name', 'prefix', 'endpoint', 'mac address', 'vni']
+
+    # Fetching data from appl_db for VNET TUNNEL ROUTES
+    vnet_rt_keys = appl_db.keys(appl_db.APPL_DB, "VNET_ROUTE_TUNNEL_TABLE*")
+    vnet_rt_keys = natsorted(vnet_rt_keys) if vnet_rt_keys else []
+
+    table = []
+    for k in vnet_rt_keys:
+        r = []
+        r.extend(k.split(":", 2)[1:])
+        val = appl_db.get_all(appl_db.APPL_DB, k)
+        r.append(val.get('endpoint'))
+        r.append(val.get('mac_address'))
+        r.append(val.get('vni'))
+        table.append(r)
+
+    click.echo(tabulate(table, header))
+
+#
+# 'vxlan' command ("show vxlan")
+#
+@cli.group(cls=AliasedGroup, default_if_no_args=False)
+def vxlan():
+    """Show vxlan related information"""
+    pass
+
+@vxlan.command()
+@click.argument('vxlan_name', required=True)
+def name(vxlan_name):
+    """Show vxlan name <vxlan_name> information"""
```

Comment:
Do you intend to have the `name `in command? Its different from the description.

Reply:
yes, the cmd is like : show vxlan name XXX, the description is not showing <>, correct it now.